presentation/src/main/java/com/kk/presentation/player/ResultRoomViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.kk.presentation.player
+
+import com.kk.presentation.baseMVI.BaseViewModel
+
+class ResultRoomViewModel :
+    BaseViewModel<ResultRoomContract.Event, ResultRoomContract.State, ResultRoomContract.Effect>() {
+    override fun createInitialState(): ResultRoomContract.State {
+        return ResultRoomContract.State(
+            ResultRoomContract.ResultRoomState.Idle
+        )
+    }
+
+    override fun handleEvent(event: ResultRoomContract.Event) {
+        when (event){
+            ResultRoomContract.Event.OnGoHomeClicked -> {goHomeView()}
```

Comment:
The navigation will be controlled by The navigation graph

Reply:
fixed in new push